src/components/custom/review/review.tsx

```diff
@@ -39,6 +45,40 @@ export const Review = (props: IGenericCustomElementProps<IReviewSchema>) => {
 	// =========================================================================
 	// RENDER FUNCTIONS
 	// =========================================================================
+	if (schema.variant === "accodion") {
+		const { items, topSection, bottomSection, button, title, expanded, collapsible, ...otherSchema } = schema;
+		return (
+			<BoxContainer
+				id={id}
+				data-testid={id}
+				title={title}
+				expanded={expanded}
+				collapsible={collapsible}
+				callToActionComponent={
+					<Button.Default
+						styleType="light"
+						type="button"
+						onClick={() => {
+							dispatchFieldEvent("button-click", id);
+						}}
+					>
+						{button?.label ?? "Edit"}
+					</Button.Default>
+				}
+			>
+				<UneditableSectionCustom
+					{...otherSchema}
+					id={id}
```

Comment:
id is meant to be unique so it should not be applied to both BoxContainer and UneditableSection, can remove it.

Reply:
remove applying id